client/cody/src/completions/index.ts

```diff
@@ -221,13 +221,13 @@ export class CodyCompletionItemProvider implements vscode.InlineCompletionItemPr
             return []
         }
 
-        logEvent('CodyVSCodeExtension:completion:started', LOG_INLINE, LOG_INLINE)
+        await logEvent('CodyVSCodeExtension:completion:started', LOG_INLINE, LOG_INLINE)
```

Comment:
Do we need to await the event sending everywhere? This introduces additional latency to some of the highly interacted with code paths for something that is not needed for the user to know about. 

If there are issues with out-of-order events we should implement proper queueing on the event logger instance instead.

```suggestion
        logEvent('CodyVSCodeExtension:completion:started', LOG_INLINE, LOG_INLINE)
```

Reply:
I'll do some testing on removing that--logEvent is making an API call that wasn't getting completed that I'm trying to solve for